dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -242,6 +242,34 @@ control dash_ingress(inout headers_t hdr,
         }
     }
 
+    action set_src_tag(tag_map_t tag_map) {
+        meta.src_tag_map = tag_map;
+    }
+
+    @name("src_tag|dash_tag")
+    table src_tag {
+        key = {
+            meta.src_ip_addr : lpm @name("meta.src_ip_addr:sip");
```

Comment:
match type: prefix_list

Reply:
Could you have a scenario where eg. 10.0.0.0/8 -> tag1, 10.1.0.0/16 -> tag2.. How do you expect lpm match here to resolve and assign tags?